src/main/java/controller/LottoController.java

```diff
@@ -0,0 +1,54 @@
+package controller;
+
+import domain.Buyer;
+import domain.LottoMatch;
+import domain.Money;
+import domain.WinningLotto;
+import java.util.HashMap;
+import util.InputParser;
+import util.Validator;
+import view.InputView;
+import view.OutputView;
+
+public class LottoController {
+
+  private InputView inputView;
+  private OutputView outputView;
+
+  public LottoController(InputView inputView, OutputView outputView) {
+    this.inputView = inputView;
+    this.outputView = outputView;
+  }
+
+  public void run() {
+    Money money = creaeteMoney();
+
+    Buyer buyer = new Buyer(money);
+    outputView.displayLottos(money.calculateTotalLotto(), buyer.createResult());
+
+    WinningLotto winningLotto = createWinningLotto();
+    HashMap<LottoMatch, Integer> lottoResult = buyer.countLottos(winningLotto);
+    outputView.displayResult(lottoResult);
```

Comment:
특별한 이유가 없다면 인터페이스를 사용하는 것이 좋습니다. :) 

```suggestion
    Map<LottoMatch, Integer> lottoResult = buyer.countLottos(winningLotto);
    outputView.displayResult(lottoResult);
```

Reply:
섬세한 리뷰 감사합니다!! 저또한 인지하고 못하고 있었습니다!  앞으로 조심해서 실수를 줄이도록 해보겠습니다 😅